exporter/datadogexporter/config.go

```diff
@@ -242,6 +242,19 @@ type TracesConfig struct {
 	flushInterval float64
 }
 
+// LogsConfig defines logs exporter specific configuration
+type LogsConfig struct {
+	// TCPAddr.Endpoint is the host of the Datadog intake server to send logs to.
+	// If unset, the value is obtained from the Site.
+	confignet.TCPAddr `mapstructure:",squash"`
+
+	// SendLogRecordBody when enabled would send string representation of body field of log record as message to Datadog
+	// By default this is disabled , as we want to parse the body string into attributes
+	// Set this to true, if we are unable to parse body or we want use datadog backend for parsing
+	// It is essential for Log Trace correlation to work, that we parse the body before reaching exporter.
+	SendLogRecordBody bool `mapstructure:"send_log_record_body"`
```

Comment:
This should be enabled by default: the majority view on the review of this PR is that the default behavior should be to always send the log record body.

Sending the body is not incompatible with asking people to parse the body and does not (AFAICT) hinder log-trace correlation. It's okay to have an option to disable sending the body if it's beneficial in some setups, but the default should be to send it.

Reply:
If parse the body and also send the body , we would be doubling the egres for the client. Am not that strongly opinionated on this, i can the default to true. But we document the behavior same. Customers might be surprised if the egres bill increases.

Reply:
as discussed offline 

1. (now) We add the body by default at the Datadog exporter if available
(this sprint) When writing the recommended Collector configuration and when deploying on the demo or, we add a 2. processor that uses a [remove operator](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/remove.md) to drop the body (or the duplicate info) from the logs before they get to our exporter
3. (this quarter, the sooner the better) We open an issue on the contrib repository asking to remove the duplicate information from the payloads